scripts/setup.sh

```diff
@@ -225,13 +225,20 @@ if [ $no_dependencies -eq 0 ]; then
     sudo="/usr/bin/sudo"
     check_if_exists "$sudo"
 
+    # debian based build_requirements
+    if [ \( "$os" == "ubuntu" -o "$os" == "debian" \) ]; then
+        build_requirements='python-dev build-essential libffi-dev libssl-dev'
+	else
```

Comment:
the indentation here looks a bit wonky. Looks like you used tabs where the rest of the file used spaces. Can you switch this over to spaces?